packages/odyssey-design-tokens/src/transition.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I checked this out from https://github.com/okta/odyssey/pull/1253; will update if a conflict emerges.